openvdb/openvdb/tree/InternalNode.h

```diff
@@ -274,9 +274,18 @@ class InternalNode
     /// Set the transient data value.
     void setTransientData(Index32 transientData) { mTransientData = transientData; }
 
+#if OPENVDB_ABI_VERSION_NUMBER >= 12
```

Comment:
I think it might be cleaner if we can avoid the ABI logic in RootNode/InternalNode/LeafNode by only providing the Index64 version (regardless of ABI) and then doing a `static_cast<Index32>` in the virtual Tree methods that use the older ABI?